### PR TITLE
Sean / feat: add duplicate landing pages for PPC campaign

### PR DESCRIPTION
**Pull request by sean-binary**
Changes:

-   Added duplicate landing pages for DMT5 and signup with copy changes
-   Factored out content from numbers component
-   Remove binary section for sign up landing page

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-new.js [hunk 1]
```diff
@@ -352,6 +355,7 @@ SignupNew.propTypes = {
     handleSocialSignup: PropTypes.func,
     handleValidation: PropTypes.func,
     is_submitting: PropTypes.bool,
+    landing: PropTypes.bool,
```
Comment (AbdallaMourad) [inline src/components/custom/_signup-new.js:358]: should we rename this variable to indicate it contains a boolean value

File: src/pages/dmt5/index.js [hunk 2]
```diff
@@ -24,6 +24,15 @@ const query = graphql`
...
 `
 
+const numbers_content = {
+    title_1: <Localize translate_text="330K+" />,
+    subtitle_1: <Localize translate_text="clients on DMT5" />,
+    title_2: <Localize translate_text="100+" />,
+    subtitle_2: <Localize translate_text="tradable assets" />,
+    title_3: <Localize translate_text="24/7" />,
+    subtitle_3: <Localize translate_text="trading" />,
+}
```
Comment (AbdallaMourad) [inline src/pages/dmt5/index.js:34]: We can re-write this as 
```js
const numbers_content = [
    {
        title: ...
        subtitle: ...
    },
    {
        title: ...
        subtitle: ...
    },
    ...
]
```

File: src/pages/dmt5/_numbers.js [hunk 3]
```diff
@@ -69,33 +69,38 @@ const Number = styled(Flex)`
...
+                        {numbers_content.title_2}
                     </StyledHeader>
                     <Splitter />
-                    <StyledText>{localize('tradable assets')}</StyledText>
+                    <StyledText>{numbers_content.subtitle_2}</StyledText>
                 </Number>
                 <Number>
                     <StyledHeader as="h2" type="page-title">
-                        {localize('24/7')}
+                        {numbers_content.title_3}
```
Comment (AbdallaMourad) [inline src/pages/dmt5/_numbers.js:92]: can we use 
```js
{numbers_content.map((content, index) => (
    <Number key={index} />
        ....
    <Number>
))}
```

File: src/pages/landing/signup/index.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        <Layout type="static" margin_top={'0'}>
+            <SEO
+                title={localize('Easy And Free Sign Up | Online Trading | Deriv.com')}
+                description={localize(
+                    'Signup to Deriv.com and trade online with as little as $1 USD on major currencies, stocks, indices, and commodities.',
+                )}
+                no_index
+            />
+            <Wrapper>
+                {!(submit_state === 'success') && (
```
Comment (AbdallaMourad) [inline src/pages/landing/signup/index.js:80]:
```suggestion
                {submit_state !== 'success' && (
```

File: src/pages/landing/dmt5/index.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    }
+`
+const numbers_content = {
+    title_1: <Localize translate_text="20+" />,
+    subtitle_1: <Localize translate_text="years of experience" />,
+    title_2: <Localize translate_text="100+" />,
+    subtitle_2: <Localize translate_text="tradable assets" />,
+    title_3: <Localize translate_text="330K+" />,
+    subtitle_3: <Localize translate_text="clients on DMT5" />,
+}
```
Comment (AbdallaMourad) [inline src/pages/landing/dmt5/index.js:33]: same here

File: src/pages/dmt5/_dhero.js [hunk 6]
```diff
@@ -270,14 +279,17 @@ const DHero = ({
...
                     <HeroHeader as="h1" type="display-title">
                         {content}
                     </HeroHeader>
                 </HeroContent>
                 <LinkWrapper>
                     {join_us_for_free && (
-                        <StyledLinkButton secondary="true" to="/signup/">
+                        <StyledLinkButton
+                            secondary="true"
+                            to={is_ppc ? '/landing/signup/' : '/signup'}
```
Comment (yashim-deriv) [inline src/pages/dmt5/_dhero.js:291]: Should we add `/`  or simply remove trailing slash for consistency?

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**